server/rest/annotation.py

```diff
@@ -160,6 +162,32 @@ def createAnnotation(self, item, params):
                     exc.args, ))
 
     @describeRoute(
+        Description('Copy an annotation from one item to an other.')
+        .param('id', 'The ID of the annotation.', paramType='path',
+               required=True)
+        .param('itemId', 'The ID of the destination item.',
+               required=True)
+        .errorResponse('Write access was denied for the item.', 403)
+        .errorResponse('Validation Error: JSON doesn\'t follow schema.')
+    )
+    @access.user
+    @loadmodel(model='annotation', plugin='large_image')
+    # @loadmodel(map={'itemId': 'item'}, model='item', level=AccessType.WRITE)
+    @filtermodel(model='annotation', plugin='large_image')
+    def copyAnnotation(self, annotation, params):
+        itemId = params['itemId']
+        user = self.getCurrentUser()
+        Item().load(
+            annotation.get('itemId'),
+            user=user,
+            level=AccessType.READ
+        )
+        item = Item().load(itemId, user=user, level=AccessType.WRITE, exc=True)
+        self.model('annotation', 'large_image').createAnnotation(
+            item, user, annotation)
```

Comment:
`annotation` is the Girder annotation resource.  `createAnnotation` takes the annotation schema object, which is `annotation['annotation']`.